api/views.py

```diff
@@ -416,3 +418,20 @@ class Accounts(generics.RetrieveAPIView):
     def get(self, request):
         serializer = AccountSerializer(request.user)
         return response.Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ContactUs(generics.CreateAPIView):
+    serializer_class = ContactUsSerializer
+    permission_clsses = permissions.AllowAny
+
+    def post(self, request):
+        serializer = self.serializer_class(data=request.data)
+        return response.Response(serializer.data, status=status.HTTP_201_CREATED)
```

Comment:
```suggestion
```
This method is redundant for the same reason it is redundant in the feedback view